lib/ui/chat_widget.dart

```diff
@@ -48,7 +50,8 @@ class _ChatWidgetState extends State<ChatWidget> {
 
   @override
   Widget build(BuildContext context) => Scaffold(
-    appBar: AppBar(
+    appBar: TopBar(title: 'Chat Support'),
+    /*appBar: AppBar(
```

Comment:
You can remove this commented out block entirely.
The original version lives in the git history, so we can review it if we need to there.